src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -397,9 +432,9 @@ export default defineComponent({
       }
     },
 
-    parseVideoData: function () {
+    parseVideoData: async function () {
       this.id = this.data.videoId
-      this.title = this.data.title
+      this.title = (await this.getDeArrowTitle()) ?? this.data.title
```

Comment:
`getDeArrowDataEntry` can return `null`
```suggestion
      this.title = (await this.getDeArrowDataEntry()?.title) ?? this.data.title
```